doc/authors.md

```diff
@@ -56,6 +57,7 @@ paveljanik (2)
 mruddy (2)
 kpcyrd (2)
 ca333 (2)
+arielgabizon (2)
```

Comment:
I see the fix in the `author_aliases`, but that change hasn't effected this file. There are a few other examples of this that existed previously Duke Leto (4)/Jonathan "Duke" Leto (12), str4d (2)/Jack Grigg (850). It might be nice to merge these all together later. This is non-blocking.